packages/h/httplib/xmake.lua

```diff
@@ -0,0 +1,42 @@
+package("httplib")
```

Comment:
This already exists: https://github.com/xmake-io/xmake-repo/blob/master/packages/c/cpp-httplib/xmake.lua

Reply:
yeah, this is a duplicate package, please use cpp-httplib directly. @fasiondog